index.html

```diff
@@ -0,0 +1,31 @@
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+    />
+    <link rel="stylesheet" href="https://unpkg.com/leaflet/dist/leaflet.css" />
+    <script src="https://unpkg.com/leaflet/dist/leaflet.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <div>
+        <h3>John Doe</h3>
+      </div>
+      <nav>
+        <a href="#projects">Projects</a>
```

Comment:
Lets use semantic html for navigation <ul> <li> tags. This makes the navigation more accessible as well.

Reply:
I assumed <nav> is a semantic html tag? Did you mean that John Doe should've also been in the nav?

Reply:
Fixed this comment too! I meant `<ul><li></li></ul>`